egs/wsj/s5/steps/score_kaldi.sh

```diff
@@ -137,9 +137,89 @@ if [ $stage -le 1 ]; then
   fi
 fi
 
+if $cer && [ $stage -le 2 ] ; then
```

Comment:
Since this is a special case, I don't like having such a big block.
How about calling steps/scoring/score_kaldi_cer.sh, and moving steps/score_kaldi.sh to steps/scoring/ and making it a link?  [trying to move to subdirs in steps/]